### PR TITLE
[Update] ELK v8.x analytic configurations

### PR DESCRIPTION
**Pull request by Avarjana**
$subject

### CODE REVIEW

File: logstash/logstash-http.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 input {
 	http {
-		host => "0.0.0.0" # default: 0.0.0.0
-		port => 8081 # default: 8080
+		host => "0.0.0.0" 
+		port => 8081
```
Comment (IsankaSR) [inline logstash/logstash-http.conf]: add a comment for the http port configuration

File: logstash/logstash-filebeat.conf [hunk 2]
```diff
@@ -111,27 +121,27 @@ output {
 	if [eventType] == "auth"{
 		elasticsearch {
 			hosts => ["https://localhost:9200"]
-			cacert => "/Users/avarjana/Documents/Work/ELK/logstash-8.1.2/config/certs/http_ca.crt"
+			cacert => "<Logstash HOME>/config/certs/http_ca.crt"
 			user => "<Elastic Username>"
```
Comment (IsankaSR) [inline logstash/logstash-filebeat.conf]: Use block capitals; <ELASTIC_USERNAME>, check other places as well

**Review by IsankaSR — APPROVED** [review]

File: kibana/saved-objects/kibana-8-x-alerts.ndjson [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
+{"excludedObjects":[],"excludedObjectsCount":0,"exportedCount":14,"missingRefCount":0,"missingReferences":[]}
```
Comment (bhagyasakalanka) [inline kibana/saved-objects/kibana-8-x-alerts.ndjson:15]: shall we add a newline

File: kibana/saved-objects/kibana-8-x-auth-and-session.ndjson [hunk 4]
```diff
@@ -1,34 +1,35 @@
...
+{"excludedObjects":[],"excludedObjectsCount":0,"exportedCount":34,"missingRefCount":0,"missingReferences":[]}
```
Comment (bhagyasakalanka) [inline kibana/saved-objects/kibana-8-x-auth-and-session.ndjson:35]: shall we add a new line. look for other places as well

File: kibana/saved-objects/kibana-7-10.ndjson [hunk 5]
```diff
@@ -1,25 +0,0 @@
...
-{"exportedCount":24,"missingRefCount":0,"missingReferences":[]}
```
[the remark this answers is not in the document]
Comment (Avarjana) [reply kibana/saved-objects/kibana-7-10.ndjson:25]: This file is removed and updated to 8.x version.

File: kibana/saved-objects/kibana-8-x-alerts.ndjson (hunk 3 above)
Comment (Avarjana) [reply kibana/saved-objects/kibana-8-x-alerts.ndjson:15]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14

File: kibana/saved-objects/kibana-8-x-auth-and-session.ndjson (hunk 4 above)
Comment (Avarjana) [reply kibana/saved-objects/kibana-8-x-auth-and-session.ndjson:35]: Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14

**Pull request merged**